improvement: ZENKO-432 update to nodejs 8 lts

rahulreddy opened the pull request.

rahulreddy committed:
improvement: ZENKO-432 update to nodejs 8 lts

bert-e commented:
# Hello rahulreddy,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get
information on this process.





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/ZENKO-886-update-nodejs` into
`development/8.0`
* [w/8.1/improvement/ZENKO-886-update-nodejs](https://github.com/scality/s3/compare/development/8.1...w/8.1/improvement/ZENKO-886-update-nodejs) will be merged into `development/8.1` (pull request #1459)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:



* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @dora-korpar

* @RachedBenMustapha

* @rahulreddy

bert-e commented:
# History mismatch



Merge commit #9ab88010ed8e06c0cc35cf1efba37061d10fa83d on the integration branch
`w/8.1/improvement/ZENKO-886-update-nodejs` is merging a branch which is neither the current
branch `improvement/ZENKO-886-update-nodejs` nor the development branch
`development/8.1`.

It is likely due to a rebase of the branch `improvement/ZENKO-886-update-nodejs` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

rahulreddy commented:
@bert-e reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/ZENKO-886-update-nodejs` into
`development/8.0`
* [w/8.1/improvement/ZENKO-886-update-nodejs](https://github.com/scality/s3/compare/development/8.1...w/8.1/improvement/ZENKO-886-update-nodejs) will be merged into `development/8.1` (pull request #1460)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:



* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @dora-korpar

* @RachedBenMustapha

* @rahulreddy

rahulreddy commented:
@bert-e wait

ploki commented on eve/main.yml:
```diff
@@ -132,6 +132,9 @@ stages:
       - Git: *clone
+      - ShellCommand:
+          name: keyscan
+          command: ssh-keyscan -H github.com > ~/.ssh/known_hosts
```
related?
should ```mkdir -p ~/.ssh``` first

rahulreddy replied:
I realized an easier way to get around this. Changes incoming.

tcarmet commented:
Hello, took a look into this PR and it's sounds like that there might be something I could fix/work on eve side to get it working properly (without workarounds and all) didn't had the time to work on it today, but I can take a look tomorrow. I'll let you guys decide (don't know if you're in a rush to get this merged or not.)

Cheers!

rahulreddy committed:
improvement: adjust test to nodejs 8

Mem data backend uses delete method to remove from it's list which nodejs
no longer represents as null. The test has been updated to assert the same
behavior by checking if each element is a falsey value.

rahulreddy commented:
@bert-e wait

rahulreddy committed:
[squash] retry once

tcarmet commented on package.json:
```diff
@@ -64,7 +64,7 @@
   },
   "optionalDependencies": {
-    "cdmiclient": "scality/cdmiclient#8f0c2e6"
+    "cdmiclient": "git+https://github.com/scality/cdmiclient#8f0c2e6"
```
Made the requested updates on eve to make it work with the `scality/cdmiclient#8f0c2e6` syntax.

Could you please make the change?

rahulreddy committed:
[squash] revert cdmiclient change
